src/main/resources/entity-types/finance/drv_fund_details.json5

```diff
@@ -0,0 +1,294 @@
+{
+  id: '4c63c7f5-1a28-4394-b401-98c02c31492d',
+  name: 'drv_fund_details',
+  private: false,
+  fromClause: 'src_finance_fund as fnd',
+  columns: [
+    {
+      name: 'id',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      isIdColumn: true,
+      valueGetter: "fnd.id"
+    },
+    {
+      name: '_version',
+      dataType: {
+        dataType: 'integerType',
+      },
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: "fnd.jsonb ->> '_version'"
```

Comment:
Didn't see that this was an integer earlier. This will need to change to an int type, which you should be able to do in a similar way to floats